lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -3019,6 +3019,7 @@ LogicalResult FIRRTLLowering::visitDecl(MemOp op) {
   if (op.getDataType()
           .cast<FIRRTLBaseType>()
           .getPassiveType()
+          .getAnonymousType()
```

Comment:
*blinks a few times* uhhhh could this entire thing just be `op.getDataType().isa<BundleType>`? (although please move to the llvm::isa form if that's what you're doing as part of this PR elsewhere, ideally that would be broken out to a separate "migrate to preferred LLVM casting methods" PR, but won't insist but don't do it only partially please).

Reply:
Yeah, `type_isa<BudnleType>(op.getDataType())` should be sufficient. Probably I messed up in applying mechanical changes.